src/controllers/organization.controller.ts

```diff
@@ -1,13 +1,15 @@
 import OrganizationService from '@/services/organization.service';
 import { NextFunction, Request, Response } from 'express';
+const url = require('url');
 
 class OrganizationController {
   public organizationService = new OrganizationService();
 
   public getOrganizationData = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
     try {
-      const orgId = parseInt(req.params.id);
-      const orgData = await this.organizationService.getOrganizationData(orgId);
+      const orgId = req.params.id;
+      const queryObject = url.parse(req.url, true).query;
```

Comment:
What does this do? `url.parse(req.url, true).query;`. Do we need to use the `url` library?

Reply:
Its just a way I found to extract query parameters

Reply:
Changed it

Reply:
Also checked out the failing tests and it was stuff that I did not touch, but might've broke when Jack changed the database - anyways, it is fixed now